### PR TITLE
Shorten the HandleHWCSettings function

### PR DESCRIPTION
**Pull request by yhe39**
Split the function to sub-function.

Change-Id: I1d32836636124dd45c31a0bdd17413ee39448a31
Tracked-On: https://jira.devtools.intel.com/browse/OAM-76800
Tests: Compile sucessful for Android, all display mode works well.
Signed-off-by: HeYue <yue.he@intel.com>

### CODE REVIEW

**Comment by yhe39** [comment]
@Shao-Feng @js0701 please help to review. Thanks!

File: public/gpudevice.h [hunk 1]
```diff
@@ -129,6 +129,38 @@ class GpuDevice : public HWCThread {
...
+  void ParseCloneDisplaySetting(
+      std::string& value, std::vector<std::vector<uint32_t>>& cloned_displays);
+  void ParsePhysicalDisplayRotation(
+      std::string& value, std::vector<uint32_t>& display_rotation,
+      std::vector<uint32_t>& rotation_display_index);
+  void ParseFloatDisplaySetting(std::string& value,
+                                std::vector<HwcRect<int32_t>>& float_displays,
+                                std::vector<uint32_t>& float_display_indices);
+
+  void DisplayInit(std::vector<uint32_t>& physical_displays,
```
Comment (YuanjunHuang) [inline public/gpudevice.h]: the Naming should be InitializeXxxx, same as ParseXxxx. as above.

Comment (yhe39) [reply public/gpudevice.h]: Thanks yuanjun, i refer to the Wan Shuang's:
#ifdef ENABLE_PANORAMA
  if (use_panorama && !use_mosaic && !use_cloned && !use_float) {
    **PanoramaInit**(total_displays_, temp_displays, panorama_displays,
                 panorama_sos_displays, available_displays);
  }
#endif

Does all the init function need change to InitializeXXX? Including the PanoramaInit？

Comment (YuanjunHuang) [reply public/gpudevice.h]: yes we shall better align with the same naming style.

Comment (yhe39) [reply public/gpudevice.h]: ok, thanks!

File: common/core/gpudevice.cpp [hunk 2]
```diff
@@ -307,6 +307,435 @@ void GpuDevice::PanoramaInit(
...
+  // TODO remove hard code
+  if (float_rect.size() == 4) {
+    float_display_indices.emplace_back(index);
+    HwcRect<int32_t> rect = HwcRect<int32_t>(
+        float_rect.at(0), float_rect.at(1), float_rect.at(2), float_rect.at(3));
+    float_displays.emplace_back(rect);
+  }
+}
+
+void GpuDevice::InitializeDisplay(std::vector<uint32_t> &physical_displays,
```
Comment (js0701) [inline common/core/gpudevice.cpp]: Can we change the name?
IntializeDisplayOrder?

File: common/core/gpudevice.cpp [hunk 3]
```diff
@@ -307,6 +307,435 @@ void GpuDevice::PanoramaInit(
...
+      displays.emplace_back(un_connected_displays.at(i));
+    }
+  }
+
+  for (size_t i = 0; i < size; i++) {
+    displays.at(i)->SetDisplayOrder(i);
+  }
+}
+void GpuDevice::InitializeRotationDisplay(
+    std::vector<uint32_t> &display_rotation,
```
Comment (js0701) [inline common/core/gpudevice.cpp]: InitializeDisplayRotation

File: common/core/gpudevice.cpp (hunk 2 above)
Comment (js0701) [reply common/core/gpudevice.cpp]: Or IntiailizeDisplayIndex

**Comment by js0701** [comment]
Thanks for working on this

File: common/core/gpudevice.cpp (hunk 2 above)
Comment (yhe39) [reply common/core/gpudevice.cpp]: Sure, will change accordingly.

**Pull request merged**